pkg/models/measurement.go

```diff
@@ -0,0 +1,35 @@
+package models
+
+import "github.com/grafana/grafana-plugin-sdk-go/data"
+
+// NOTE:  this likely shoud go in the Plugin SDK since it will be useful from plugins
+
+// Measurement is a single measurement value
+type Measurement struct {
+	Name   string                      `json:"name,omitempty"`
+	Time   int64                       `json:"time,omitempty"`   // units are usually ms, but depend on the channel
+	Values map[string]interface{}      `json:"values,omitempty"` // typically number or string
+	Config map[string]data.FieldConfig `json:"config,omitempty"` // optional list of field configs
+	Labels map[string]string           `json:"labels,omitempty"` // labels are applied to all values
```

Comment:
The comments should go above and be on sentence form. When we enable better linting, it will catch this.